bridge/README.md

```diff
@@ -0,0 +1,39 @@
+# Bridge SDK
+
+## Contents
+
+- [Compatibility](#compatibility)
+- [Versioning](#versioning)
+
+## Compatibility
+
+- **Minimum SDK**: 28
+- **Target SDK**: 34
+- **Device Types Supported**: Phone and Tablet
```

Comment:
I know this was taken from the other README but I'm wondering if we should remove these. There shouldn't be any UI here so maybe the SDK values are enough.